src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -844,19 +845,25 @@ void SpatialPooler::inhibitColumnsGlobal_(const vector<Real> &overlaps,
   // faster than a regular sort because it stops after it partitions the
   // elements about the Nth element, with all elements on their correct side of
   // the Nth element.
-  std::nth_element(
+  tNth.start();
+  std::nth_element(htm::parallel::mode,
```

Comment:
inhibition identified as one of the slowest methods in SP. 
`nth_element` as the most significant in global inh.